### ISSUE TITLE
NemoWindow .nemo-inactive-pane .view - does not work in Icon view

### ISSUE THREAD

**Issue by smurphos** [linuxmint/nemo#1592]
```
 * Nemo version  3.4.7
 * Is issue with desktop or windowed nemo? Both
 * Mint 18.2, Cinnamon 3.4.6
 * 2nd Generation Core Processor Family Integrated Graphics Controller, driver=i915 
 * 64 bit
 ```

NemoWindow .nemo-inactive-pane .view { background-color: #xxxxxx; } in gtk3.css is only applied to list and compact views. It doesn't apply to a pane in icon view.

**Steps to reproduce**

Apply any GTK3 theme that includes a different background for nemo-inactive-pane. E.g Mint-Y or Mint-X. Open Nemo. Open an extra pane. Click between the panes. Change views between Icon, List and Compact on one or both panes and observe the different behaviour on clicking between panes.

**Expected behaviour**

All three views should react in the same way.

**Other information**

**Comment by JosephMcc** [comment]
Cannot confirm this. It seems to work fine for me in Mint-Y.

**Comment by smurphos** [comment]
This is what I see with Mint Y

1)Left hand pane has focus, right hand pane has a darker background.

![screenshot from 2017-08-29 06-27-24](https://user-images.githubusercontent.com/29017677/29805710-474d42da-8c83-11e7-91fe-5c231220aedd.png)

2)Right hand pane has focus, no darker background on left hand pane in icon view

![screenshot from 2017-08-29 06-27-03](https://user-images.githubusercontent.com/29017677/29805704-3eb5c7f0-8c83-11e7-93c4-b2e38a66cd34.png)

3)Right hand pane has focus, darker background on left hand pane in list view

![screenshot from 2017-08-29 06-27-45](https://user-images.githubusercontent.com/29017677/29805709-474a4242-8c83-11e7-95dc-2004e811765f.png)

There is a colour change in icon view but it applies to the last focused icon and it's label which might not necessarily be visible in the pane. In these screenshots right has focus and then left.

![screenshot from 2017-08-29 06-37-43](https://user-images.githubusercontent.com/29017677/29805957-9b56f9f6-8c84-11e7-8235-b9d929158a66.png)
![screenshot from 2017-08-29 06-37-37](https://user-images.githubusercontent.com/29017677/29805958-9b5705a4-8c84-11e7-9349-d86ec68659c8.png)

**Comment by smurphos** [comment]
Just an additional note - this issue persists for me on a VM with Mint 18.3 upgraded to GTK3.22.28 via PPA and with a GTK3.20+ theme including the css below. In this scenario list view works fine but neither icon nor compact views display any color change. 
'''
 .nemo-window .nemo-inactive-pane .view, .nemo-window .nemo-inactive-pane iconview {
    background-color: #20323b;
    color: #daeaf2; }
'''

**Comment by smurphos** [comment]
Issue appears to be solved with Cinnamon 3.8.4+ in both Mint 19 beta and Manjaro in a VB on Mint 19 host..
